scripts/test-verify-phase-51-closeout-evaluation.sh

```
@@ -0,0 +1,90 @@
+
+absolute_path_repo="${workdir}/absolute-path"
+copy_valid_repo "${absolute_path_repo}"
+mac_home_prefix="/""Users/example"
+printf 'Run %s/Dev/codex-supervisor/dist/index.js.\n' "${mac_home_prefix}" >>"${absolute_path_repo}/docs/phase-51-closeout-evaluation.md"
+assert_fails_with \
+  "${absolute_path_repo}" \
+  "Forbidden Phase 51 closeout evaluation: workstation-local absolute path detected"
+
+windows_backslash_path_repo="${workdir}/windows-backslash-path"
+copy_valid_repo "${windows_backslash_path_repo}"
+windows_backslash_path="$(printf 'C:%bUsers%bexample%brepo%bfile.md' '\\' '\\' '\\' '\\')"
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)